build/package-artifacts.yml

```diff
@@ -45,15 +45,15 @@ steps:
 
   - template: package-templates.yml
 
-# Publish artifacts
+# Generate SBOM
+  - task: ManifestGeneratorTask@0
+    displayName: 'SBOM Manifest Generator'
+    inputs:
+      BuildDropPath: '$(Build.ArtifactStagingDirectory)'

```

Comment:
I see '/bin' is being missed here. I'm fine if it was intentional and tested.

Reply:
yeah it was intentional